cpp/tests/grouped_rolling/grouped_rolling_test.cpp

```diff
@@ -655,12 +758,83 @@ class GroupedTimeRangeRollingTest : public cudf::test::BaseFixture {
     return col.release();
   }
 
+  std::unique_ptr<cudf::column> create_row_number_reference_output(
+    cudf::column_view const& timestamp_column,
```

Comment:
Would be useful to have several tests that verify this function itself spits out the right thing.   That is, compare the output of this to some known hardcoded input/output values.

Reply:
This part is a little clumsy, at the moment. I've been trying to follow the conventions of `rolling_test`, from which the structure was lifted. I do use hardcoded values in the [JNI tests for the same API](https://github.com/rapidsai/cudf/pull/4881/files#diff-102fccf0687a5edd9689e08de91fdc3cR1788). Is that more palatable?

I could add further tests in a follow-on, if you advise it.

Reply:
That's fine.  Make sure to add an issue for it.